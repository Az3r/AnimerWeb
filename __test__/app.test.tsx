import React from 'react';
import { render } from '@testing-library/react';
import { Typography } from '@material-ui/core';

test('should App tree not changed', () => {
  const { container } = render(<Typography>Hello</Typography>);
  expect(container.firstChild).toMatchSnapshot();
});
