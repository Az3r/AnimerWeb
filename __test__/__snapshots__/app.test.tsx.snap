// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should App tree not changed 1`] = `
<p
  class="MuiTypography-root MuiTypography-body1"
>
  Hello
</p>
`;
